src/wasm2asm.h

```diff
@@ -1624,6 +1527,10 @@ Ref Wasm2AsmBuilder::processFunctionBody(Function* func, IString result) {
             case LtFloat32:
             case LtFloat64:
               return ValueBuilder::makeBinary(left, LT, right);
+            case RotLInt32:
+            case RotRInt32:
+              std::cerr << "should be removed already" << std::endl;
+              abort();
```

Comment:
aborts should be `WASM_UNREACHABLE()`, although we can leave to later and fix up the other places it needs to be updated.